docs/js/example.js

```diff
@@ -1,14 +1,125 @@
-var x = new XMLHttpRequest();
+/* global VANILLA_VERSION */
+(function() {
+  if (!window.VANILLA_VERSION) {
+    throw Error('VANILLA_VERSION not specified.');
+  }
+
+  var CODEPEN_PREFILL_CONFIG = {
+    title: 'Vanilla example',
+    head: "<meta name='viewport' content='width=device-width, initial-scale=1'>",
+    stylesheets: [
+      // link to latest Vanilla CSS
+      'https://assets.ubuntu.com/v1/vanilla-framework-version-' + VANILLA_VERSION + '.min.css',
```

Comment:
This would mean there was no way to check the docs locally with local examples and framework?

Reply:
This applies to CodePen embeds only, so yes, inside the docs the CodePen examples will always use latest released version of Vanilla, not the local one.

The example pages (https://vanilla-framework-canonical-web-and-design-pr-2705.run.demo.haus/examples/) use the /build.css as before, so local changes can still be viewed on them, Percy makes screenshots of them, etc.

So the only "downside" is that the examples embedded in docs use released version, not development one.
This on the other hand may be seen as an advantage, thanks to that we can deploy the docs site in between Vanilla releases, while being sure embedded examples are not affected by any local changes to SCSS. (But of course the example pages themselves would be still affected).

We probably could try to pass current /build.css instead of released version.
But, there may be a problem with local development, as passing localhost URL may have some issues when embedded in CodePen.